src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { FooterMenu, FooterMenuItemType } from './FooterMenu';
+import { graphql, useStaticQuery } from 'gatsby';
+import { FooterSocial } from './FooterSocial';
+import { FooterLogoAndStatus } from './FooterLogoAndStatus';
+
+export const Footer = () => {
+  const data = useStaticQuery(graphql`
+    query FooterMenuQuery {
+      menuData: pageFurnitureYaml(name: { eq: "FooterMenu" }) {
+        items {
+          label
+          link
+          items {
+            label
+            link
+          }
+        }
+      }
+      ablySocial: pageFurnitureYaml(name: { eq: "AblySocial" }) {
+        items {
+          label
+          link
+        }
+      }
+      bottomMenu: pageFurnitureYaml(name: { eq: "FooterBottomMenu" }) {
+        items {
+          label
+          link
+        }
+      }
+    }
+  `);
+
+  const menuDataItems = data.menuData.items;
+  const ablySocialLinks = data.ablySocial.items;
+  const bottomLinks = data.bottomMenu.items;
```

Comment:
if you do `data.bottomMenu.items ?? [];` for all of these, I think you won't have to check inline to see if the items exist before mapping later?